### PR TITLE
Adding EverShop to the E-commerce software list

### PR DESCRIPTION
**Pull request by treoden**
Adding EverShop to the E-commerce software list

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/evershop.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+website_url: https://evershop.io/
+description: EverShop is a NodeJS ecommerce platform with essential commerce features. Built with NodeJS and React. Modular architecture and fully customizable.
+licenses:
+  - GPL-3.0
+platforms:
+  - Nodejs
+tags:
+  - E-commerce
+source_code_url: https://github.com/evershopcommerce/evershop
+demo_url: https://demo.evershop.io/
```
Comment (kokomo123) [inline software/evershop.yml:11]:
```suggestion
demo_url: https://demo.evershop.io/
depends_3rdparty: true
```
I propose adding this flag as due to paypal, stripe is out of the user's control.

File: software/evershop.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: EverShop
+website_url: https://evershop.io/
+description: EverShop is a NodeJS ecommerce platform with essential commerce features. Built with NodeJS and React. Modular architecture and fully customizable.
+licenses:
+  - GPL-3.0
+platforms:
+  - Nodejs
```
Comment (Rabenherz112) [inline software/evershop.yml:7]:
```suggestion
platforms:
  - Docker
  - Nodejs
```

Also [supports Docker](https://evershop.io/docs/development/getting-started/installation-guide#install-evershop-using-docker) as an installation method

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]

File: software/evershop.yml (hunk 2 above)
Comment (treoden) [reply software/evershop.yml]: Thanks. Updated

**Review by Rabenherz112 — APPROVED** [review]

File: software/evershop.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: EverShop
+website_url: https://evershop.io/
+description: EverShop is a NodeJS ecommerce platform with essential commerce features. Built with NodeJS and React. Modular architecture and fully customizable.
```
Comment (nodiscc) [inline software/evershop.yml:3]:
```suggestion
description: E-commerce platform with essential commerce features. Modular architecture and fully customizable.
```

shorter description

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by treoden** [comment]
Hi,

Thank you for your reviews , are we waiting for some thing else?

Thanks

**Comment by Ki-er** [comment]
>  You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

We appreciate your patience, however this will be merged whenever Nodiscc is available!

**Pull request merged**